lua/entities/gmod_wire_expression2/core/number.lua

```diff
@@ -408,6 +431,10 @@ __e2setcost(2) -- approximation
 local deg2rad = pi / 180
 local rad2deg = 180 / pi
 
+registerFunction("huge", "", "n", function(self, args)
```

Comment:
No legitimate uses that I could name off the top of my head, though when I wrote that I thought of string functions and the use of callbacks. Say in a case where you passed a function that returns a number, this would easily allow pi to be used. Of course that could be done with a UDF, or just adding a special case, as for why I said convenient.